cdsctl monitoring

on Exit:

```
panic: reflect: call of reflect.Value.NumField on zero Value

goroutine 1 [running]:
reflect.flag.mustBe(0x0, 0x19)
	/Users/yesnault/go/src/reflect/value.go:201 +0xc8
reflect.Value.NumField(0x0, 0x0, 0x0, 0x0)
	/Users/yesnault/go/src/reflect/value.go:1170 +0x34
github.com/ovh/cds/cli.listItem(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42067c060, 0xf)
	/Users/yesnault/src/github.com/ovh/cds/cli/cobra.go:347 +0x194
github.com/ovh/cds/cli.newCommand.func1(0xc420204480, 0xc4201f1480, 0x0, 0x2)
	/Users/yesnault/src/github.com/ovh/cds/cli/cobra.go:187 +0x2685
github.com/ovh/cds/vendor/github.com/spf13/cobra.(*Command).execute(0xc420204480, 0xc4201f1420, 0x2, 0x2, 0xc420204480, 0xc4201f1420)
	/Users/yesnault/src/github.com/ovh/cds/vendor/github.com/spf13/cobra/command.go:750 +0x2c1
github.com/ovh/cds/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4202046c0, 0x46b6500, 0x1, 0xc4201f1380)
	/Users/yesnault/src/github.com/ovh/cds/vendor/github.com/spf13/cobra/command.go:831 +0x30e
github.com/ovh/cds/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4202046c0, 0xc4201f1380, 0x471c762)
	/Users/yesnault/src/github.com/ovh/cds/vendor/github.com/spf13/cobra/command.go:784 +0x2b
main.main()
	/Users/yesnault/src/github.com/ovh/cds/cli/cdsctl/main.go:69 +0x8a5
```

--- a/cli/cobra.go
+++ b/cli/cobra.go
@@ -343,6 +343,10 @@ func listItem(i interface{}, filters map[string]string, quiet bool, fields []str
 		s = reflect.ValueOf(i)
 	}
 
+	if s.Kind() != reflect.Struct {
+		return nil
+	}
+
 	t := reflect.TypeOf(i)
 	var ok = true
 	for i := 0; i < s.NumField() && ok; i++ {

